src/Mapbender/ManagerBundle/Resources/public/components.js

```diff
@@ -22,6 +22,45 @@ $(function() {
         return false;
     });
 
+    $(document).on('click', ".sortByColumn", function(){
+        var $this = $(this);
+        var asc = true;
+
+        $this.css('text-decoration', 'underline');
+        $this.siblings().css('text-decoration', 'none');
+        $this.siblings().find('span').hide();
+
+        if($this.find('span').is(":hidden")){
+            $this.find('span').removeClass('fa-arrow-down').addClass('fa-arrow-up').show();
```

Comment:
Similar to my comment in #1632, here design and functionality is very tightly coupled which makes customisations more complicated. A solution to decouple it would be to have two spans instead of one with the classes `sort-indicator-asc` and `sort-indicator-desc` (or similar, in addition to the fa-classes) and only toggle their visibility. That way only css and/or the twig template needs to be modified to change the appearence.

Reply:
I actually have to toggle between 3 classes: `fa fa-sort`, `fa fa-sort-amount-asc` and `fa fa-sort-amount-desc`. I couldn't think of a technically nice solution how to toggle with 3 classes (or 3 spans) without making the source code more complicated.

Reply:
alright :+1: